examples/Replay

```diff
@@ -0,0 +1,46 @@
+#include <Servo.h>
+#include <EEPROM.h>
+
+Servo myServo;        // create servo object to control a servo
+
+bool record = false;  // boolean to determine if recording is taking plce
+int potpin = 0;       // analog pin used to connect the potentiometer
+int val;              // variable to read the value from the analog pin
+int read;             // variable to read the data from memory
+
+void setup() {
+  pinMode(13, OUTPUT); // button on pin 2
+  myServo.attach(9);   // attaches the servo on pin 9 to the servo object
+
+  Serial.begin(9600);
+  digitalWrite(13, HIGH);
+}
+
+void loop() {
+  // begin recording on potentiometer if button is pressed 
+  if (record == true) {
+    for (int i = 1; i <= EEPROM.length(); i++) {
```

Comment:
```suggestion
    for (int i = 1; i < EEPROM.length(); i++) {
```
EEPROM addresses are zero-indexed, so the highest address is EEPROM.length() - 1.